pkg/status/status_apiserver.go

```diff
@@ -30,3 +31,14 @@ func getLeaderElectionDetails() map[string]string {
 	leaderElectionStats["status"] = "Running"
 	return leaderElectionStats
 }
+
+func getDCAStatus() map[string]string {
+	clusterAgentDetails := make(map[string]string)
+	dcaCl, err := clusteragent.GetClusterAgentClient()
+	if err != nil {
+		clusterAgentDetails["Error"] = err.Error()
+		return clusterAgentDetails
+	}
+	clusterAgentDetails["Endpoint"] = dcaCl.ClusterAgentAPIEndpoint
```

Comment:
you don't need the client to get `ClusterAgentAPIEndpoint` it's a public package variable no?